Library/ViewModels/LiveStreamContainerViewModel.swift

```diff
@@ -0,0 +1,247 @@
+import KsApi
+import LiveStream
+import ReactiveSwift
+import ReactiveExtensions
+import Result
+import Prelude
+
+public protocol LiveStreamContainerViewModelType {
+  var inputs: LiveStreamContainerViewModelInputs { get }
+  var outputs: LiveStreamContainerViewModelOutputs { get }
+}
+
+public protocol LiveStreamContainerViewModelInputs {
+  func configureWith(project: Project, event: LiveStreamEvent?)
+  func closeButtonTapped()
+  func liveStreamViewControllerStateChanged(state: LiveStreamViewControllerState)
+  func retrievedLiveStreamEvent(event: LiveStreamEvent)
+  func viewDidLoad()
```

Comment:
we like to add doc comments for all inputs and outputs. if you have time to add them for all view models, that'd be great, but a future PR would be fine too :)